src/pages/patientView/therapyRecommendation/FollowUpTable.tsx

```diff
@@ -0,0 +1,337 @@
+        super(props);
+        this.state = {
+            followUps: props.followUps,
+            deletions: props.deletions,
+        };
+    }
+
+    @computed
+    get columnWidths() {
+        return {
+            [ColumnKey.INFO]: ColumnWidth.INFO,
+            [ColumnKey.THERAPYRECOMMENDATION]:
+                this.props.containerWidth - ColumnWidth.INFO,
+        };
+    }
+
+    @observable selectedTherapyRecommendation:
+        | ITherapyRecommendation
+        | undefined;
+    @observable backupTherapyRecommendation: ITherapyRecommendation | undefined;
+    @observable showFollowUpForm: boolean;
+
+    private _columns = [
+        {
+            name: ColumnKey.INFO,
+            render: (followUp: IFollowUp) => (
+                <div>
+                    <span className={styles.edit}>
+                        <DefaultTooltip
+                            trigger={['hover', 'focus']}
+                            overlay={getTooltipAuthorContent(
+                                'Follow-up',
+                                followUp.author
+                            )}
+                            destroyTooltipOnHide={false}
+                        >
+                            <i className={'fa fa-user-circle'}></i>
+                        </DefaultTooltip>
+                    </span>
+                    <input
+                        type="date"
+                        value={followUp.date}
+                        style={{
+                            display: 'block',
+                            marginTop: '2px',
+                            marginBottom: '2px',
+                        }}
+                        onChange={(e: React.FormEvent<HTMLInputElement>) => {
+                            const newDate = e.currentTarget.value;
+                            const newFollowUps = this.state.followUps.slice();
+                            newFollowUps.find(
+                                x => x.id === followUp.id
+                            )!.date = newDate;
+                            this.setState({ followUps: newFollowUps });
+                        }}
+                    ></input>
+                    <LabeledCheckbox
+                        checked={followUp.therapyRecommendationRealized}
+                        onChange={() => {
+                            const newTrr = !followUp.therapyRecommendationRealized;
+                            const newFollowUps = this.state.followUps.slice();
+                            newFollowUps.find(
+                                x => x.id === followUp.id
+                            )!.therapyRecommendationRealized = newTrr;
+                            this.setState({ followUps: newFollowUps });
+                        }}
+                        labelProps={{ style: { marginRight: 10 } }}
+                    >
+                        <span style={{ marginTop: '-2px' }}>Realized</span>
+                    </LabeledCheckbox>
```

Comment:
@nr23730 do you maybe have a good idea how to make these checkboxes reusable for the tumor response criteria?

Reply:
You mean, so that other stuff will only be displayed if this is checked?

Reply:
> You mean, so that other stuff will only be displayed if this is checked?

That's not an issue, we can do that with a <Collapsible>. 
What I mean is an idea for implementing the checkboxes ideally without copy&pasting this block 12 times.

Reply:
> That's not an issue, we can do that with a 

My tag disappeared. I meant a Collapsible.

> What I mean is an idea for implementing the checkboxes ideally without copy&pasting this block 12 times.

We should also think about if all of them should be checkboxes. Some of the response criteria are mutually exclusive, right?

Reply:
Definitely! I will see if there is anything on the FHIR side and our documentation sheet so that might be able to reuse something. We could then discuss this next Wednesday after our Jour-fixe.